src/modules/scraper/adapter/messaging/BlocksEventsConsumer.ts

```diff
@@ -63,8 +68,20 @@ export class BlocksEventsConsumer {
           throw error;
         }
       }
+
+      try {
+        await this.insertRawDepositEvent(event);
+      } catch (error) {
+        if (error instanceof QueryFailedError && error.driverError?.code === "23505") {
+          // Ignore duplicate key value violates unique constraint error.
+          this.logger.warn(error);
+        } else {
+          throw error;
+        }
+      }
```

Comment:
after the deposit events are fetched for a particular block range, save them in the DB